scripts/tools/rl_kube_warmup.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
```

Comment:
Deleted this file as it was part of other PR

Reply:
Perfect!